spec/rubocop/cli/cli_auto_gen_config_spec.rb

```diff
@@ -588,7 +588,7 @@ def a; end
           .to eq(<<-YAML.strip_indent)
             # Offense count: 2
             # Cop supports --auto-correct.
-            # Configuration parameters: EnforcedStyle.
+            # Configuration parameters: EnforcedStyle, AutoCorrect.
```

Comment:
I think "AutoCorrect" is not a configuration parameter. We can omit "AutoCorrect" from here, and change ToDos like the following.

```yaml
# Offense count: 2
# Cop supports --auto-correct (disabled by default).
# Configuration parameters: EnforcedStyle.
...
```

I'll fix it on another pull-request.